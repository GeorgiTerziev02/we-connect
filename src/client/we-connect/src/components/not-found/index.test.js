import React from 'react'
import renderer from 'react-test-renderer'
import NotFound from './index'
import TestingEnvironment from '../../test-utils/router'

// always mock before describe
jest.mock('../../images/banner_error_404.jpg', () => 'Image')

describe('Not found component', () => {
    it('should render not found component', () => {
        const tree = renderer.create(
            <TestingEnvironment value={{
                user: {
                    id: 2342
                },
                loggedIn: true
            }}>
                <NotFound />
            </TestingEnvironment>
        ).toJSON()

        expect(tree).toMatchSnapshot()
    })
})