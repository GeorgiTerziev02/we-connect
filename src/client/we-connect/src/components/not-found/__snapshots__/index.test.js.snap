// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found component should render not found component 1`] = `
<div
  className="container"
>
  <img
    className="img-fluid"
    src="Image"
  />
  <div
    className="container"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <button
        className="btn btn-primary"
        disabled={false}
        type="button"
      >
        Go to posts feed
      </button>
    </a>
  </div>
</div>
`;
